shared/router-v2/router.desktop.js

```diff
@@ -278,6 +284,7 @@ const styles = Styles.styleSheetCreate({
   }),
   modalContainer: {
     ...Styles.globalStyles.fillAbsolute,
+    zIndex: 99999,
```

Comment:
can we avoid this? this is problematic for modals that are supposed to have floating stuff on top of them.

Reply:
we should discuss a solution. on profile there was some absolutely positioned stuff due to the section list which goes above the modal